### PR TITLE
add express gulp es6

### PR DESCRIPTION
**Pull request by festinalente91**

### CODE REVIEW

**Comment by ace0625** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/recbook/recbook-server/pull-request/3/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=recbook/recbook-server)

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```
Comment (ace0625) [inline .babelrc]: new line needed

File: .gitignore [hunk 2]
```diff
@@ -38,3 +38,6 @@ jspm_packages
 
 # Webstrom
 .idea/
+
+# Gulp
+dist/
```
Comment (ace0625) [inline .gitignore]: New line needed

File: gulpfile.babel.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+gulp.task('clean', () => rimraf.sync(SOURCE.DIST));
+
+gulp.task('build', ['clean'], () => {
+  return gulp.src(SOURCE.ALL)
+    .pipe(sourcemaps.init())
+    .pipe(babel())
+    .pipe(sourcemaps.write("."))
+    .pipe(gulp.dest("dist"))
+});
```
Comment (ochanje210) [inline gulpfile.babel.js]: new line needed?

**Pull request merged**